R/Data-class.R

```diff
@@ -5,7 +5,7 @@ NULL
 
 # GeneralData-class ----
 
-#' `GeneralData`
+#' `GeneralData` A Class for General Data Input
```

Comment:
I think we also don't have more in the title for the other data classes, so don't need that here then either?